pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js

```diff
@@ -54,6 +54,11 @@ router.get('/', function(req, res, next) {
                 if (ERR(err, next)) return;
                 res.locals.assessment_text_templated = assessment_text_templated;
 
+                res.locals.htmlTitle = res.locals.assessment_set.abbreviation
+                  + res.locals.assessment.number
+                  + ': ' + res.locals.assessment.title
+                  + ' - ' + res.locals.course.short_name;
+
```

Comment:
I think `title` might be a bit much here? It's often quite long.

Should we indicate that these are PrairieLearn pages anywhere in the title? We could end titles with `(PrairieLearn)` or `(PL)`, for example.